airgun/views/host_new.py

```diff
@@ -63,35 +77,145 @@ class Overview(Tab):
 
         @View.nested
         class DetailsCard(Card):
-            ROOT = '(.//article[contains(@class, "pf-c-card")])[1]'
+            ROOT = './/article[.//div[text()="Details"]]'
             details = HostDetailsCard()
 
         @View.nested
         class HostStatusCard(Card):
-            ROOT = '(//article[contains(@class, "pf-c-card")])[2]'
+            ROOT = './/article[.//span[text()="Host status"]]'
             status = Text('.//h4[contains(@data-ouia-component-id, "OUIA-Generated-Title")]')
 
-            status_success = Text('.//span[@class="status-success"]')
-            status_warning = Text('.//span[@class="status-warning"]')
-            status_error = Text('.//span[@class="status-error"]')
-            status_disabled = Text('.//span[@class="disabled"]')
+            status_success = Text('.//span[span[@class="status-success"]]')
+            status_warning = Text('.//span[span[@class="status-warning"]]')
+            status_error = Text('.//span[span[@class="status-error"]]')
+            status_disabled = Text('.//span[span[@class="disabled"]]')
+
+        @View.nested
+        class InstallableErrataCard(Card):
+            ROOT = './/article[.//div[text()="Installable errata"]]'
+
+            security_advisory = Text('.//a[contains(@href, "type=security")]')
+            bug_fixes = Text('.//a[contains(@href, "type=bugfix")]')
+            enhancements = Text('.//a[contains(@href, "type=enhancement")]')
+
+        @View.nested
+        class TotalRisksCard(Card):
+            ROOT = './/article[.//div[text()="Total risks"]]'
+
+            low = Text('.//*[@id="legend-labels-0"]/*')
+            moderate = Text('.//*[@id="legend-labels-1"]/*')
+            important = Text('.//*[@id="legend-labels-2"]/*')
+            critical = Text('.//*[@id="legend-labels-3"]/*')
 
         @View.nested
         class RecentJobsCard(Card):
-            ROOT = '(//article[contains(@class, "pf-c-card")])[5]'
+            ROOT = './/article[.//div[text()="Recent jobs"]]'
             is_table_loaded = './/ul[@aria-label="recent-jobs-table"]'
 
     @View.nested
     class Content(Tab):
-        pass
+        ROOT = './/div'
```

Comment:
I'd suggest not setting ROOT for tab widgets. Its overriding the ParametrizedLocators that use TAB_NAME (or the class name), and is likely causing the behavior where you're having to manually select tabs.

Reply:
I will try it, thanks for the suggestion :+1:

Reply:
We did some debugging this morning, and found that the sub-tabs here can't resolve their elements when nested. A workaround is to override `Tab.child_widget_accesssed` on the subtab classes, and first call `self.parent.select()` before calling `self.select()`.

Reply:
@mshriver As the workaround does not work, are you okay with using the ROOT?

Reply:
Adding BZ link and TODO